### PR TITLE
add severity to logs and update testify closes #2589

### LINKED ISSUE #2589: Add severity level to logs
This is useful for knowing, for example, if you've encountered any errors during a test run of Lantern.

### PR DESCRIPTION
**Pull request by myleshorton**
OK, here's another stab at this @xiam. I actually did not use gost get for the new stretchr package, but I figure we should just move to gb sooner rather than later and bring in all versioned dependencies at that time.

### CODE REVIEW

File: src/github.com/getlantern/golog/golog_test.go [hunk 1]
```diff
@@ -6,26 +6,34 @@ import (
 	"io/ioutil"
 	"os"
 	"regexp"
+	"strings"
 	"testing"
 	"time"
 
-	"github.com/getlantern/testify/assert"
```
Comment (xiam) [inline src/github.com/getlantern/golog/golog_test.go]: Looks like we initially forked this one in order to add a small change https://github.com/stretchr/testify/compare/master...getlantern:master.

The change was just adding more debugging to the output of a test, so it's OK to use the upstream package. We should probable deprecate `github.com/getlantern/testify` too if we don't plan to make major changes.

**Comment by xiam** [comment]
This one looks better, thanks @myleshorton!

File: src/github.com/getlantern/golog/golog_test.go (hunk 1 above)
Comment (myleshorton) [inline src/github.com/getlantern/golog/golog_test.go]: Yeah exactly @xiam. Ours was actually worse at displaying more standard test output, though, which is what we need for this particular test, so I reverted. I agree we should probably just remove getlantern/testify, as I'm also guessing similar functionality is now upstream.

**Comment by myleshorton** [comment]
Thanks @xiam!

**Pull request merged**